app/backend/src/couchers/helpers/clusters.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+
+from geoalchemy2.shape import from_shape
+
+from couchers.models import Cluster, ClusterRole, ClusterSubscription, Node, Page, PageType, PageVersion, Thread
+
+DEFAULT_PAGE_CONTENT = "There is nothing here yet..."
+DEFAULT_PAGE_TITLE_TEMPLATE = "Main page for the {name} {type}"
+
+
+def create_node(session, geom, parent_node_id):
+    node = Node(geom=from_shape(geom), parent_node_id=parent_node_id)
+    session.add(node)
+    session.commit()
```

Comment:
You shouldn't `commit` here. You can `flush` if you need to flush the SQL to the db e.g. to get the node's ID (autogenerated by db).

Committing will commit the SQL transaction which might lead to an inconsistent state if you commit halfway through creating an object that consists of multiple db objects (e.g. an orphan node without a main page).